SOPT_Seminar_Task/SOPT_Seminar_Task/FisrtViewController.swift

```diff
@@ -0,0 +1,289 @@
+        $0.font = UIFont(name: "Pretendard-SemiBold", size: 15)
+        $0.placeholder = "비밀번호"
+        $0.isSecureTextEntry = true
+        $0.autocapitalizationType = .none   //시작 소문자로
+        $0.setPlaceholderColor(.gray)
+        $0.addLeftPadding()
+        
+    }
+    lazy var eyeButton = UIButton().then
+    {
+        $0.setImage(UIImage(named: "hiePwEye"), for: .normal)
+        $0.addTarget(self, action: #selector(eyeButtonTapped), for:.touchUpInside)
+        
+    }
+    
+    private lazy var loginButton = UIButton().then{
+        $0.setTitle("로그인하기", for: .normal)
+        $0.setTitleColor(.lightGray, for: .normal)
+        $0.backgroundColor = .black
+        $0.layer.borderColor = UIColor.gray.cgColor
+        $0.layer.borderWidth = 1
+        $0.titleLabel?.font = UIFont.systemFont(ofSize: 14)
+        $0.contentHorizontalAlignment = .center
+        $0.isEnabled = false
+        
+        $0.addTarget(self, action: #selector(pushButtonTapped), for: .touchUpInside)
+        
+    }
+    
+    private lazy var findIdButton = UIButton().then{
+        $0.setTitle("아이디찾기", for: .normal)
+        $0.setTitleColor(.lightGray, for: .normal)
+        $0.backgroundColor = .black
+        $0.titleLabel?.font = UIFont.systemFont(ofSize: 14)
+        $0.contentHorizontalAlignment = .center
+    }
+    
+    private lazy var findPwButton = UIButton().then{
+        $0.setTitle("비밀번호찾기", for: .normal)
+        $0.setTitleColor(.lightGray, for: .normal)
+        $0.backgroundColor = .black
+        $0.titleLabel?.font = UIFont.systemFont(ofSize: 14)
+        $0.contentHorizontalAlignment = .center
+    }
+    
+    private let middleLabel = UILabel().then
+    {
+        $0.text = "|"
+        $0.textColor = UIColor(red: 0.18, green: 0.18, blue: 0.18, alpha: 1)
+    }
+    
+    private let askNoId = UILabel().then
+    {
+        $0.text = "아직 계정이 없으신가요?"
+        $0.textColor = UIColor(red: 0.384, green: 0.384, blue: 0.384, alpha: 1)
+        $0.font = UIFont(name: "Pretendard-SemiBold", size: 14)
+        $0.textAlignment = .center
+    }
+    
+    private lazy var makeIdButton = UIButton().then
+    {
+        $0.setTitle("닉네임 만들러가기", for: .normal)
+        $0.setTitleColor(.lightGray, for: .normal)
+        $0.titleLabel?.font = UIFont.systemFont(ofSize: 16)
+        $0.setUnderline()
+        
+    }
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        idField.delegate = self
+        pwField.delegate = self
+        
+        style()
+        setLayout()
+        
+    }
+    
+    func textFieldDidBeginEditing(_ textField: UITextField) {
+        // textField 눌렀을 때, border 색상 변경
+        textField.layer.borderColor = UIColor.white.cgColor
+    }
+    
+    func textFieldDidEndEditing(_ textField: UITextField) {
+        // 다시 안눌렀을 때, 기본 border 색상으로 변경
+        textField.layer.borderColor = UIColor.gray.cgColor
+    }
```

Comment:
여기서는 반대로 editing 이 끝났을때 eyeButton 이 없어지게!
```suggestion
    } func textFieldDidEndEditing(_ textField: UITextField) {
        eyeButton.isHidden = true
    }
```